gcs/CHANGES.md

```diff
@@ -64,6 +64,13 @@
     fs.gs.auth.refresh.token
     ```
 
+1.  Merge all output stream types functionality in the default output stream,
+    and remove obsolete property:
```

Comment:
Please add which of the original 3 types is the new default now.

Reply:
Done.